packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3863,7 +3863,7 @@ class Flex extends MultiChildRenderObjectWidget {
     this.crossAxisAlignment = CrossAxisAlignment.center,
     this.textDirection,
     this.verticalDirection = VerticalDirection.down,
-    this.textBaseline,
+    this.textBaseline = TextBaseline.alphabetic,
```

Comment:
Can you update the documentation of this property to include what the default value is?

Reply:
Done